addons/mail/static/src/core/web/message_actions_patch.js

```diff
@@ -14,7 +14,8 @@ messageActionsRegistry
     })
     .add("forward", {
         condition: (component) =>
-            component.props.message.canReplyAllandForward(component.props.thread),
+            component.props.message.canReplyAllandForward(component.props.thread) ||
+            component.props.message.is_note,
```

Comment:
Shouldn't we have canReplyAll and canFoward then ? Strange to have condition computed in two places, better have the computation done right directly.